manifests/n/nkdAgility/AzureDevOpsExportActionableAgile/0.1.1/nkdAgility.AzureDevOpsExportActionableAgile.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: nkdAgility.AzureDevOpsExportActionableAgile
+PackageVersion: 0.1.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: AzureDevOps.Export.ActionableAgile.ConsoleUI.exe
+  PortableCommandAlias: AzureDevOpsExport
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.DotNet.Runtime.8
Installers:
```

(Automated response - build 869.)

---

Is a dependancy the right thing to add when its a portable package?